README.md

```diff
@@ -1,10 +1,13 @@
-
-![logo](docs/source/images/SkyPilot-logo-wide.png)
+<p align="center">
+  <img src="docs/source/images/SkyPilot-logo-wide.png" alt="SkyPilot" width=80%/>
+</p>
 
 ![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
 [![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
 
-SkyPilot is a framework to run any workload seamlessly across different cloud providers through a unified interface. No knowledge of cloud offerings is required or expected – you simply define the workload and its resource requirements, and SkyPilot will automatically execute it on AWS, Google Cloud Platform or Microsoft Azure.
+SkyPilot is a framework to run machine learning[^1] workloads seamlessly across different cloud providers through a unified interface. No knowledge of cloud offerings is required or expected – you simply define the workload and its resource requirements, and SkyPilot will automatically execute it on AWS, Google Cloud Platform or Microsoft Azure.
```

Comment:
Can we change this first line to match our docs
```
SkyPilot is a framework for easily running machine learning projects on any cloud through a unified interface.
```
(with the footnote added)?  A few words shorter. "workloads" vs "projects": either sounds good to me.